pyramid_oereb/core/config.py

```diff
@@ -384,7 +423,11 @@ def _read_availabilities():
             availability_config.get('source').get('class'),
             **availability_config.get('source').get('params')
         )
-        return availability_reader.read()
+
+        avaliablity_records = availability_reader.read()
+        if len(avaliablity_records) == 0:
```

Comment:
I think the availability table can be empty if all topics are available in all municipalities.